RocketChat/Rocket.Chat.ReactNative
# Chore: Evaluate RoomMembersView - TypeScript

## Proposed changes


## Issue(s)	


## How to test or reproduce


- Inside a room
- Go to Room Actions View and click on Members
- Then, you can click in members, should appear a bottom sheet with some buttons

## Screenshots

## Types of changes




## Checklist



## Further comments
